### PR TITLE
Blockly.Extensions.buildTooltipForDropdown(..): Deferred validation.

### PR DESCRIPTION
**Pull request by AnmAtAnm**
Defer tooltip message string check until after load, when all Blockly.Msg should be loaded.

Also, fix annotation error.

### CODE REVIEW

File: core/utils.js [hunk 1]
```diff
@@ -834,3 +834,28 @@ Blockly.utils.insertAfter_ = function(newNode, refNode) {
...
+/**
+ * Calls a function after the page has loaded, possibly immediately.
+ *
+ * This function does not run in a Node.js environment due to lack of document
+ * object.
+ * @param {function()} fn Function to run.
+ * @throws Error Will throw if no global document can be found (e.g., Node.js).
+ */
+Blockly.utils.runAfterPageLoad = function(fn) {
+  if (document == null) {
```
Comment (rachel-fenichel) [inline core/utils.js]: Why not if (!document) ?

File: core/utils.js [hunk 2]
```diff
@@ -834,3 +834,28 @@ Blockly.utils.insertAfter_ = function(newNode, refNode) {
     parentNode.appendChild(newNode);
   }
 };
+
+/**
+ * Calls a function after the page has loaded, possibly immediately.
+ *
+ * This function does not run in a Node.js environment due to lack of document
```
Comment (rachel-fenichel) [inline core/utils.js]: This comment ("This function does not run...) belongs at line 92 of extensions.js, with a note that in that case validation will not occur until the first block of that type is constructed.

The fact that it'll break in node.js is covered by the @throws annotation.

File: core/utils.js (hunk 1 above)
Comment (AnmAtAnm) [inline core/utils.js]: Done.

File: core/utils.js (hunk 2 above)
Comment (AnmAtAnm) [inline core/utils.js]: Done.

**Pull request merged**